src/os/bluestore/BlueStore.cc

```diff
@@ -6011,10 +6011,22 @@ int BlueStore::_fsck(bool deep, bool repair)
         }
       );
       if (intersects) {
-	derr << "fsck error: free extent 0x" << std::hex << offset
-	     << "~" << length << std::dec
-	     << " intersects allocated blocks" << dendl;
-	++errors;
+	if (offset == SUPER_RESERVED &&
+	    length == min_alloc_size - SUPER_RESERVED) {
```

Comment:
We'd better confirm that min_alloc_size is greater than SUPER_RESERVED here since min_alloc_size is configurable